src/NumericalAlgorithms/FiniteDifference/MonotisedCentral.hpp

```diff
@@ -0,0 +1,95 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+#include <cmath>
+#include <cstddef>
+#include <utility>
+
+#include "NumericalAlgorithms/FiniteDifference/Reconstruct.hpp"
+#include "Utilities/ForceInline.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/Math.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+template <size_t Dim>
+class Direction;
+template <size_t Dim>
+class Index;
+/// \endcond
+
+namespace fd::reconstruction {
+namespace detail {
+struct MonotisedCentralReconstructor {
+  SPECTRE_ALWAYS_INLINE static std::array<double, 2> pointwise(
+      const double* const q, const int stride) noexcept {
+    using std::abs;
+    using std::min;
+    // NOLINTNEXTLINE(cppcoreguidelines-pro-bounds-pointer-arithmetic)
+    const double a = q[stride] - q[0];
+    // NOLINTNEXTLINE(cppcoreguidelines-pro-bounds-pointer-arithmetic)
+    const double b = q[0] - q[-stride];
+    const double slope = 0.5 * (sgn(a) + sgn(b)) *
+                         min(0.5 * abs(a + b), 2.0 * min(abs(a), abs(b)));
+    // NOLINTNEXTLINE(cppcoreguidelines-pro-bounds-pointer-arithmetic)
+    return {{q[0] - 0.5 * slope, q[0] + 0.5 * slope}};
+  }
+
+  SPECTRE_ALWAYS_INLINE static constexpr size_t stencil_width() noexcept {
+    return 3;
+  }
+};
+}  // namespace detail
+
+/*!
+ * \ingroup FiniteDifferenceGroup
+ * \brief Performs monotised central-difference reconstruction on the `vars` in
+ * each direction.
+ *
+ * The monotised central-difference reconstruction \cite RezzollaBook first
+ * computes:
+ *
+ * \f{align}
+ * \sigma^n_j\equiv \textrm{minmod}
+ *           \left(\frac{u^n_{j+1} - u^n_{j-1}}{2\Delta x},
+ *                 2\frac{u^n_j-u^n_{j-1}}{\Delta x},
+ *                 2\frac{u^n_{j+1}-u^n_j}{\Delta x}\right)
+ * \f}
+ *
+ * where
+ *
+ * \f{align}
+ *  \mathrm{minmod}(a,b,c)=
+ * \left\{
+ *   \begin{array}{ll}
+ *    \mathrm{sgn}(a)\min(\lvert a\rvert, \lvert b\rvert, \lvert c\rvert)
+ *      & \mathrm{if} \; \mathrm{sgn}(a)=\mathrm{sgn}(b)=\mathrm{sgn}(c) \\
+ *    0 & \mathrm{otherwise}
+ *   \end{array}\right.
+ * \f}
+ *
+ * The reconstructed solution in the \f$i\f$th cell is given by
+ *
+ * \f{align}
+ * u^n_j(x)=u^n_j + \sigma^n_j (x-x_j)
```

Comment:
Please define x_j
Please define the difference between "u^n_j(x)" and "u^n_j"... perhaps just by editing the pre-equation sentence so it reads "The reconstructed solution u^n_j(x) in..." (and relying on an earlier defn of u^n_j as I asked for before).